README.md

```diff
@@ -113,45 +119,15 @@ kubectl delete cluster "${CLUSTER_NAME:-"test1"}" -n metal3
 
 ### Deploying with Tilt
 
-It is possible to use Tilt to run the CAPI, BMO and CAPM3 components. For this, run:
+It is possible to use Tilt to run the CAPI, BMO and CAPM3 components. Tilt
+ephemeral cluster will utilize Kind and Docker, so it requires a machine with
+`IMAGE_OS=ubuntu`. For this, run:
```

Comment:
```suggestion
`IMAGE_OS=ubuntu`. 
Once the above requirements fulfilled, run:
```

Reply:
I'm not really following what you want changes here. This section of the document starts here, so there are no "above requirements" you propose, besides requirement of running it on Ubuntu and exporting `IMAGE_OS=ubuntu`.

I'll reword it a bit, maybe that works better?

Reply:
Looks good, thanks!

/approve

Reply:
@furkatgofurov7 Can you change your review to approve as well, it is still showing you requesting changes, despite using prow approve keyword.